e2e/test/scenarios/dashboard-filters/parameters.cy.spec.js

```diff
@@ -592,6 +601,421 @@ describe("scenarios > dashboard > parameters", () => {
       cy.findByText("Balistreri-Ankunding").should("not.exist");
     });
   });
+
+  describe("when auto-wiring parameters across cards with matching fields", () => {
+    beforeEach(() => {
+      cy.intercept("GET", "/api/dashboard/**").as("dashboard");
+    });
+
+    describe("when wiring parameter to all cards for a filter", () => {
+      it("should automatically wire parameters to cards with matching fields", () => {
+        createDashboardWithCards([
+          {
+            card_id: ORDERS_BY_YEAR_QUESTION_ID,
+            row: 0,
+            col: 0,
+            size_x: 5,
+            size_y: 4,
+          },
+          {
+            card_id: ORDERS_COUNT_QUESTION_ID,
+            row: 0,
+            col: 4,
+            size_x: 5,
+            size_y: 4,
+          },
+        ]).then(dashboardId => {
+          visitDashboard(dashboardId);
+          cy.wait("@dashboard");
+        });
+
+        cy.icon("pencil").click();
+
+        cy.findByTestId("dashboard-header").icon("filter").click();
+        popover().within(() => {
+          cy.contains("Text or Category").click();
+          cy.findByText("Is").click();
+        });
```

Comment:
We have this helper function to quickly set filters
```suggestion
      setFilter("Text or Category", "Is");
```